dds/DCPS/Util.h

```diff
@@ -197,6 +200,10 @@ bool set_intersect(SetA& sA, const SortedB& sB, LessThan lessThan)
   return !sA.empty();
 }
 
+/// Return the number of CORBA::Longs required for the bitmap representation of
+/// sequence numbers between low and high, inclusive (maximum 8 longs).
+OpenDDS_Dcps_Export CORBA::ULong bitmap_num_longs(const SequenceNumber& low, const SequenceNumber& high);
```

Comment:
Does this need to be in the core library?  Even though it doesn't use RTPS types, it depends on the RTPS details,  Or maybe it could go with DisjointSequence which does deal with the bitmap representation.